jest.config.e2e.js

```diff
@@ -0,0 +1,6 @@
+// jest config for e2e tests.
```

Comment:
Have you considered using command-line options embedded into `package.json:scripts` instead of a separate configuration file?

Reply:
If you still prefer to use configuration files, consider placing them inside the `tests` directory.